### PR TITLE
Remove CP and DP segmentation differences and update documentation

### PR DESCRIPTION
**Pull request by roshankern**
This PR is ready for review and addresses 3 [issues](https://github.com/WayScience/IDR_stream/issues) with very small changes.

#9 - CP/DP Segmentation Differences: 
These segmentation differences were caused by using different versions of CellPose and instantiating CellPose with `models.Cellpose` instead of `models.CellposeModel`

#10 - DP Environment Installation Issues:
After installing DP, numpy needs to be upgraded with `pip install numpy==1.23.3`

#11 - Aspera Installation Instructions:
The installation instructions for Aspera have been updated to make it easier for Ubuntu users to follow along.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +164,8 @@ Clone the repository into `idrstream/` with:
 ```sh
```
Comment (axiomcura) [inline README.md]: I do not know if `sh` is the right choice. Maybe `console` may be a better choice because it can let the reader know this is done within the `console` 

`sh` assumes it's a shell script, unless it is!

File: idrstream/segment.py [hunk 2]
```diff
@@ -69,17 +69,19 @@ def get_object_locations(
...
         objects_data = []
-
-        cellpose_model = models.Cellpose(
+        
+        cellpose_model = models.CellposeModel(
             gpu=self.use_GPU, model_type=self.model_specs["model_type"]
         )
-        masks, flows, styles, diams = cellpose_model.eval(
+        
+        masks, flows, *_ = cellpose_model.eval(
```
Comment (axiomcura) [inline idrstream/segment.py:77]:
```suggestion
        masks, flows, _ = cellpose_model.eval(
```
I am not familiar with `*_` for unused variables. Conventionally `_` is used for unused variables unless this is a new update I am not aware off. 

For what I know `*` is used to unpack data structures like: `tuples` and `lists`.

Feel free to accept or reject this suggestion!

File: README.md [hunk 3]
```diff
@@ -214,17 +224,25 @@ Close the GUI and reopen to confirm the path is correct.
...
 pip install -e .
 ```
 
+Installing this version of DeepProfiler will downgrade numpy to the wrong version, so it is necessary to reinstall numpy with:
+```sh
+# make sure that the conda environment `idrstream_dp` is activated
+conda activate idrstream_dp
+pip install numpy==1.23.3
+```
+
```
Comment (axiomcura) [inline README.md]:
```suggestion
Installing this version of DeepProfiler will downgrade numpy to the wrong version, so it is necessary to reinstall numpy with:

```console
# make sure that the conda environment `idrstream_dp` is activated
conda activate idrstream_dp
pip install numpy==1.23.3
```
switched `sh` to `console`

MD formatting rules about [code blocks](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md031.md)

File: README.md [hunk 4]
```diff
@@ -214,17 +224,25 @@ Close the GUI and reopen to confirm the path is correct.
 
 # DeepProfiler Project Setup:
 
-Deep Profiler must be installed via Github.
+DeepProfiler must be installed via Github.
 Commit [`2fb3ed3`](https://github.com/cytomining/DeepProfiler/commit/2fb3ed3027cded6676b7e409687322ef67491ec7) was used while developing `idrstream`.
 Install the repository into `idrstream/` with:
 ```sh
```
Comment (axiomcura) [inline README.md]:
```suggestion
```console
```

File: README.md [hunk 5]
```diff
@@ -125,6 +125,14 @@ conda activate idrstream_dp
 
 Install Aspera high-speed transfer client as described at https://github.com/IBM/aspera-cli#installation.
```
Comment (axiomcura) [inline README.md]:
```suggestion
Install Aspera high-speed transfer client as described at <https://github.com/IBM/aspera-cli#installation>.
```
MD formatting about bare links [source](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md034.md)

**Comment by axiomcura** [comment]
Hey @roshankern great PR :muscle: . Suggested some minor formatting changes. Feel free to accept or reject them !

File: README.md [hunk 6]
```diff
@@ -125,6 +125,14 @@ conda activate idrstream_dp
...
 We used the direct installation method.
+On Ubuntu, the direct installation method is as follows:
+
+1) Install Ruby on Ubuntu: `sudo apt install ruby-full`
+2) Confirm Ruby install: `ruby --version`
+3) Install aspera-cli gem: `gem install aspera-cli`
+4) Upgrade aspera-cli gem to latest version: `gem update aspera-cli`
+5) Install acsp: `ascli conf ascp install`
+6) Confirm install and locate acsp (for step 1a): `ascli config ascp show`
```
Comment (d33bs) [inline README.md:138]: The explicit directions here are great. As an enhancement, consider converting this into a `shell` or `console` codeblock for ease of use (retaining comments on the steps from the existing numbering here). Doing this would retain the understandability and make it easy for anyone to reproduce these steps on another device. It could also prepare for containerization work, if desired or appropriate in the future.

**Review by axiomcura — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (roshankern) [reply README.md]: Implemented in https://github.com/WayScience/IDR_stream/pull/12/commits/5cf59bb6be7873572cf196ebb2babb3f015f2bb7!

File: README.md [hunk 7]
```diff
@@ -214,17 +224,25 @@ Close the GUI and reopen to confirm the path is correct.
...
-# make sure that the conda environment `idrstream_dp` is activated
 pip install -e .
 ```
 
+Installing this version of DeepProfiler will downgrade numpy to the wrong version, so it is necessary to reinstall numpy with:
+```console
+# make sure that the conda environment `idrstream_dp` is activated
+conda activate idrstream_dp
+pip install numpy==1.23.3
+```
```
Comment (d33bs) [inline README.md]: This seems like an area which could get difficult to navigate, especially as versions change or revisions are made with this repo. Is there a more elegant way to specify these environment details within the .yml environment specifications?

File: README.md (hunk 3 above)
Comment (roshankern) [reply README.md]: Thank you! Added in https://github.com/WayScience/IDR_stream/pull/12/commits/caf087552dd8b7544189c790228a63ee8f810a5a

File: README.md (hunk 4 above)
Comment (roshankern) [reply README.md]: changed in https://github.com/WayScience/IDR_stream/pull/12/commits/5cf59bb6be7873572cf196ebb2babb3f015f2bb7

**Review by d33bs — COMMENTED** [review]
Nice work! Sending along some comments in case they're helpful for future considerations. Please don't hesitate to let me know if you have any questions.

**Comment by roshankern** [comment]
Thank you @d33bs ! I will keep these comments in mind while developing in the future!
